include/obstack.h

```diff
@@ -117,11 +117,8 @@
  *   h: pointer to the handle used to grow the object.
  *   size: number of bytes
  *
- * Returned Value:
- *   The new address just after the end of the currently growing object.
- *
  ****************************************************************************/
-#define obstack_blank_fast(h, size) ((h)->next_free += (size))
+#define obstack_blank_fast(h, size) ((void)((h)->next_free += (size)))
```

Comment:
let' prepend blank line between the comment and ALL macro definition.

Reply:
Ok